lib/private/legacy/OC_Util.php

```diff
@@ -969,6 +969,13 @@ public static function checkServer(\OC\SystemConfig $config) {
 			];
 		}
 
+		if ($config->getValue('secret', '') === '') {
+			$errors[] = [
+				'error' => $l->t('The required \'secret\' config variable is not configued in the config.php file'),
```

Comment:
```suggestion
				'error' => $l->t('The required \'secret\' config variable is not configued in the config.php file.'),
```
Based on the screenshot it feels a dot is missing there given there is one on the other line.

Maybe we should have some requiredConfigVars array that lists the required vars? Is secret really the only one? Feels unlikely.